ui/src/app/components/help-sidebar.tsx

```diff
@@ -124,15 +125,15 @@ const iconStyles = {
 };
 
 interface Props {
-  location: string;
+  helpContent: string;
+  hideSidebar: number;
   profileState: any;
-  participant?: any;
-  setParticipant?: Function;
 }
 
 interface State {
   activeIcon: string;
   filteredContent: any;
+  participant: any;
```

Comment:
Nit: We should type this if possible. Maybe as `ParticipantCohortStatus` if I am understanding correctly? (And the filtered content)
Do we know what data it will have?